### ISSUE TITLE
Decrease steps for dataset 5?

### ISSUE THREAD

**Issue by KatieBelli** [ZFTurbo/Music-Source-Separation-Training#207]
> Found tracks in dataset: 582
> Total tasks to process: 76040
> 
> (...)
> 
> Created 75879 good chunks from 582 tracks
> Precomputed 75879 chunks

With my dataset, it currently takes 2 hours for each epoch (37940 steps).

Does dataset type 5 include the chosen gradient accumulation for calculating the steps? 
If not, how can we make it consider it while calculating the steps per epoch?

It looks like this in my config file:

>   batch_size: 2
>   gradient_accumulation_steps: 4

A bit off topic:
Does this type also feeds the audio synced like in dataset 4?
